pyproject.toml

```
@@ -0,0 +1,46 @@
+[build-system]
+requires = ["setuptools", "pyyaml"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+include-package-data = true
+
+[tool.setuptools.packages]
+find = {}
+
+[project]
+name = 'vcstool'
+version = '0.3.0+por'
+authors = [{name='Dirk Thomas', email='web@dirk-thomas.net'}]
+maintainers = [{name='Aaron Wood', email='aaron.wood@plusonerobotics.com'}]
+description = 'vcstool provides a command line tool to invoke vcs commands on multiple repositories'
+license='Apache-2.0'
+classifiers = [
+    'Intended Audience :: Developers',
+    'Programming Language :: Python',
+    'Topic :: Software Development :: Version Control',
+    'Topic :: Utilities'
+]
+dependencies ['pyyaml']
```

This looks like an error, should be
```suggestion
dependencies = ['pyyaml']
```
right?